Vagrantfile

```diff
@@ -69,7 +69,13 @@ Vagrant.configure(2) do |config|
   #   sudo apt-get install -y apache2
   # SHELL
   config.vm.provision "shell", path: "scripts/vagrant_setup.sh"
+  config.vm.provision "shell", inline: <<-SHELL
+    sudo apt-get update
+    sudo apt-get install -y git python python-pip
+    sudo pip install -U pip six gitpython git+https://github.com/mitodl/testinfra@python_ruby_package#egg=testinfra
+  SHELL
```

Comment:
Ah, yes, you're right. There's a script in the `scripts` directory for installing testinfra to run via py.test which is why I was getting confused. Carry on :)

We should abstract this to be cross-distro and executed as a script in the `scripts` directory and then put that in the cookiecutter.